crates/next-custom-transforms/tests/fixture/server-actions/server/26/input.js

```diff
@@ -1,5 +1,6 @@
 const noop = (action) => action
 
+// TODO: should use `log` as function name?
```

Comment:
Higher order functions are currently not covered by this PR. I assume we could handle them as well when we loosen the condition on when to assign `self.arrow_or_fn_expr_ident`. I'm not sure yet whether this will yield false positives though. Needs to be checked.